src/server.ts

```diff
@@ -1,14 +1,20 @@
 import * as express from 'express';
 import { URL } from 'url';
-import * as path from 'path'
+import * as path from 'path';
 
 import * as meetbotManager from './meetbot-manager';
 
 const HTTP_PORT = process.env.HTTP_PORT || 8080;
 const server = express();
 
 server.use(express.json());
-server.use('/ui', express.static(path.join(__dirname, './public')))
+server.use('/ui', express.static(path.join(__dirname, './public')));
+
+server.get('/', (_req, res) => {
+	res.send(
+		`Meetbot server is online! Listening for requests on port ${HTTP_PORT}`,
```

Comment:
If you reach this point you already know the port. Let's just redirect to the UI

Reply:
I agree but this has a different usecase when @edufschmidt's PR comes back. The root endpoint should be the login screen that we are going to have.